src/internal/views/nunjucks/charge-information/workflow-tabs/change-request.njk

```diff
@@ -22,7 +22,7 @@
                 </tr>
             </thead>
             <tbody>
-                {% for chargeInformationRow in licences.changeRequest %}
+                {% for chargeInformationRow in licences.changeRequest.data %}
                     <tr class="govuk-table__row">
                         <td class="govuk-table__cell">{{chargeInformationRow.licence.licenceNumber}}</td>
```

Comment:
Please consider wrapping `<td class="govuk-table__cell">{{row.licence.licenceNumber}}</td>` with a href to the licence ID